cli/src/pcluster/resources/imagebuilder/parallelcluster.yaml

```diff
@@ -204,6 +204,7 @@ phases:
                 apt-get clean
                 apt-get -y update
                 apt-get -y install build-essential curl wget jq
+                sed -i "s/1/0/g" /etc/apt/apt.conf.d/20auto-upgrades
```

Comment:
Is it possible to be more strict on matching, something like
```
"s/Unattended-Upgrade \"1\"/Unattended-Upgrade \"0\"/g"
```

Reply:
I see now that we are also changing `Update-Package-Lists`.

Reply:
I would also move this change right after the apt lock check, right after the `flock` line

Reply:
Shall we move this change under condition ${CfnParamUpdateOsAndReboot} == false? Don't we disable ubuntu automatic update in all the cases?